controllers/fingerprint.go

```diff
@@ -14,6 +15,14 @@ import (
 	"github.com/m4rw3r/uuid"
 )
 
+//FingerprintData -
```

Comment:
Leaving a comment like this is worse than leaving no comment. Delete it or provide content.